README.md

```diff
@@ -1,15 +1,18 @@
 # UI Kit
 
-> UI Kit for react-native and react-native-web apps with modal and drawer
-
-[UIKit Storybook Link](https://react-native-web-uikit.storybook.trucknet.io)
-
-![Modal/Drawer](docs/uikit_storybook.gif)
+> UI Kit for react-native and react-native-web
 
 ## StoryBook
 
-    - git clone git@github.com:trucknet-io/react-native-web-uikit.git
-    - npm install
+- git clone git@github.com:trucknet-io/react-native-web-uikit.git
+- npm install
+- for adding story name file `ComponentName.story.tsx` in component folder
+- for adding readme name file `ComponentName.story.md` in component folder
```

Comment:
README is useful for the component regardless of storybooks, it is a common way to create a `README.md` file in a folder with a description of the module.

So it is better to name them `README.md` without any attribute to storybooks.

GitHub, for exampe, will render those REAMDE, when you are in the module's folder.